### ISSUE TITLE
CFScript property parsing error

### ISSUE THREAD

**Issue by dajester2013** [cfeclipse/cfeclipse#28]
Railo (and I believe ACF 10) allows cfscript CFCs to define their properties as follows:
```coldfusion
property String MyStringProperty;
```
CFEclipse is currently reporting this as an exception.

**Comment by kwadrofonik** [comment]
I concur that this is incorrectly reported as an exception, but I would like to clarify that it is actually a missing feature. Pure cfscript components support shorthand property declarations like the above example as of CF9.

![2016-04-08_2123](https://cloud.githubusercontent.com/assets/18372344/14407505/b1635ae8-fe99-11e5-8eae-69b680a9b99d.png)

However, the more formal declaration is supported by CFEclipse as per the documentation: http://help.adobe.com/en_US/ColdFusion/9.0/Developing/WSE99A664D-44E3-44d1-92A0-5FDF8D82B55C.html#WSbfbbf049491c37b01e63e3d129fef91a7d-8000

![2016-04-08_2134](https://cloud.githubusercontent.com/assets/18372344/14407534/06335356-fe9b-11e5-85d6-30fef8c9b876.png)

**Comment by denuno** [comment]
The good news is, this should be fixed on the latest dev version.

http://cfeclipse.org/update-dev/

If it's not, then I fixed this sometime after the last code push.  I've got another code push that's just been hanging out until I have time to free some space on the build server, which updates us to the latest cfml.parsing and cflint, which I'll try to get out next weekend, which is what I'm running locally, and I don't see this problem in it, at least... so, soon, my pretties, soon, this, and a few other annoying things will be resolved (if not already by the dev build currently available).
